### PR TITLE
Fixture load scrap

### PR DESCRIPTION
**Pull request by lihaobhsfer**

### CODE REVIEW

File: src/store/modules/game.js [hunk 1]
```diff
@@ -66,7 +66,6 @@ export default {
 			if (Object.hasOwnProperty.call(newGame, 'p1Ready')) state.p1Ready = newGame.p1Ready;
 			if (Object.hasOwnProperty.call(newGame, 'passes')) state.passes = newGame.passes;
 			if (Object.hasOwnProperty.call(newGame, 'players')) state.players = _.cloneDeep(newGame.players);
-			if (Object.hasOwnProperty.call(newGame, 'srap')) state.scrap = _.cloneDeep(newGame.scrap);
```
Comment (itsalaidbacklife) [inline src/store/modules/game.js:69]: Nice -- looks like you caught a typo (`srap`). Did you mean to replace this instead of removing it?

**Review by itsalaidbacklife — CHANGES_REQUESTED** [review]
game.js should likely still update the scrap on updateGame()

File: src/store/modules/game.js (hunk 1 above)
Comment (lihaobhsfer) [reply src/store/modules/game.js:69]: There is already a line of code for scrap several lines above. Sorry, should've mentioned this earlier :)

**Review by itsalaidbacklife — APPROVED** [review]
oops my mistake; no issue!

**Pull request merged**